piton/design/include/define.h.pyv

```diff
@@ -133,6 +133,7 @@ print "`define Y_TILES %d"   % Y_TILES;
 `define MSG_SUBLINE_VECTOR_      15:12
 `define MSG_OPTIONS_2_           15:0
 `define MSG_ADDR_HI_             55
+`define MSG_ADDR_LO_             16
```

Comment:
I believe you're meant to use (MSG_ADDR_HI_ - MSG_ADDR_LEN + 1) to get this number - could you change this rather than having it hardcoded to 16?